docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Private Packagist organizations that are synchronized with a remote organization have automatically all admins and owners of the synchronization assigned to the admins and owners team in Private Packagist.
+
+If an organization is synchronized with multiple remote organizations then only admins and owners of the primary synchronization are added to the respective teams.
+
+When promoting a synchronization to a primary synchronization from a different integration (e.g. from GitHub to Bitbucket), new admins and owners will be added from the primary synchronization to Private Packagist.
+Users who are not in the new team will lose access to the organization. You can resotre their access by connecting the account to the new service from the profile page.
```

Comment:
`Users who are not in the new team will lose access to the organization.` this is something to applies in general. Not only when switching integrations

`You can resotre their access by connecting the account` typo in restore and the sentence reads as if an admin can restore access for other users. Every users needs to connect their own account.